### PR TITLE
Storage: Add the time cost about stream in local/remote

### PR DESCRIPTION
**Pull request by JaySon-Huang**
### What problem does this PR solve?

Issue Number: ref https://github.com/pingcap/tiflash/issues/8675

Problem Summary:

### Check List

Tests 

- [ ] Unit test
- [ ] Integration test
- [x] Manual test (add detailed scripts or steps below)
- [ ] No code

Side effects

- [ ] Performance regression: Consumes more CPU
- [ ] Performance regression: Consumes more Memory
- [ ] Breaking backward compatibility

Documentation

- [ ] Affects user behaviors
- [ ] Contains syntax changes
- [ ] Contains variable changes
- [ ] Contains experimental features
- [ ] Changes MySQL compatibility

### Release note

```release-note
None
```

### CODE REVIEW

**Review by CalvinNeo — APPROVED** [review]
lgtm

File: dbms/src/Flash/Statistics/TableScanImpl.cpp [hunk 1]
```diff
@@ -57,12 +65,21 @@ void TableScanStatistics::collectExtraRuntimeDetail()
             {
                 /// remote read
                 updateTableScanDetail(cop_stream->getConnectionProfileInfos());
+                // TODO: Can not get the execution time of remote read streams?
```
Comment (SeaRise) [inline dbms/src/Flash/Statistics/TableScanImpl.cpp:68]: Doesn't `cop_stream->getProfileInfo()` work?

**Pull request merged**